core/src/main/kotlin/xtdb/vector/IVectorIndirection.kt

```diff
@@ -0,0 +1,37 @@
+package xtdb.vector
```

Comment:
meta: if you're doing straight Java -> Kotlin, these are best committed separately straight to `main` :pray: